### ISSUE TITLE
UPDATE: email text for invitation to join a footlight calendar

### ISSUE THREAD

**Issue by troughc** [culturecreates/tech-support-tickets#17]
Research shows that providing more information in emails that have links (from unknown sources) helps machines understand that the email is not spam nor phishing (I checked with a anti-malware expert at Google). 

Our email invitation currently could appear to be spam / phishing because there is very little information about who this is coming from or why the recipient should click the link. This new content will help machines (and people) know that this is a legitimate email invitation to join a calendar.

**English text:**

[insert name of inviter organization] is inviting you to participate in the [insert name of calendar] calendar. 

[name of calendar, for example Tout culture] is powered by Footlight, a flexible way for people to list events that are easy to find and available for reuse. With Footlight, your events are structured for discoverability and shared for reuse by the broader arts and culture community via Artsdata. You can also import events that have already been created to minimize repetitive data entry.  

[blue button] Accept invitation

Should you need any assistance, please contact [insert calendar contact email].

You are receiving this email because an administrator would like you to contribute to their calendar. This email was sent to [insert email address of email recipient].

**French text:**

[Nom de l'invitant organization] vous invite à participer au calendrier [nom de calendrier].

[name of calendar, for example Tout culture] est propulsé par Footlight, un moyen flexible pour les utilisateurs de répertorier des événements qui sont faciles à trouver et disponibles pour la réutilisation. Avec Footlight, vos événements sont structurés pour être découverts et partagés et peuvent être réutilisés par l'ensemble de la communauté artistique et culturelle via Artsdata. Vous pouvez également importer des événements qui ont déjà été créés afin de minimiser la saisie de données répétitives.

[bouton bleu] Accepter l'invitation

Si vous avez besoin d'aide, veuillez contacter [insérer le courriel du contact du calendrier].

Vous recevez ce message car l'administrateur souhaite que vous contribuiez à son calendrier. Ce courriel a été envoyé à [insérer le courriel du destinataire de courriel].

NOTE:

You can also find the new text (in French and english) in the Footlight CMS copy spreadsheet here: https://docs.google.com/spreadsheets/d/1Wh2a-ZHVs94L7JPrgxv-sWRFm6_m7tfXSdPIln0GZiI/edit
(email copy tab, row 11)

The layout can be like this mockup:

![Image](https://user-images.githubusercontent.com/64489312/244152923-24a3e877-14c8-4e6b-ad20-44a9aad4f51f.png)

**Comment by troughc** [comment]
@dev-aravind the new text for the invitation email has been reviewed and approved in both english and french. Please update the copy in the backend as discussed. We need this to be ready for Thursday June 15, morning our time 8:30 AM at the latest. If we can have it a day earlier I could do a test run before we send out the email invitations for Tout culture.
Thank you so much!

cc @saumier

**Comment by saumier** [comment]
@dev-aravind Please take up this issue as a priority Tuesday. I think we should do a test email to verify before sending out the bulk invites on Thursday morning. So if we deploy the latest backend Wednesday morning in our maintenance window during 1 am - 4 am ET then we can do tests during the day Wednesday.

**Comment by dev-aravind** [comment]
@troughc

Is the text at the bottom ("You are receiving this email ...") expected to be in Italics?
When are we planning to implement Welcome emails? Estimate : 1 day

**Comment by dev-aravind** [comment]
Please see the attachment. 

![Image](https://github.com/culturecreates/tech-support-tickets/assets/132546883/5f10fed5-1cd9-445a-b5e3-0fb3378211cf)
![Image](https://github.com/culturecreates/tech-support-tickets/assets/132546883/eecf7242-a176-4c53-8999-25113741919b)

**Comment by troughc** [comment]
Tested on Chrome french and english
Everything is great except for one thing: the email contact address for Signe Laval. 
Right now it is set to [support@culturecreates.com](mailto:support@culturecreates.com).
The email contact should be: [support@culturelaval.ca](mailto:support@culturelaval.ca). I am wondering why it would have changed @saumier @dev-aravind ?
dUPDATE: never mind answering my question - just realized it is because it is in staging not in production. Sorry!

I will fix and test again...

example image of the email I got (english version). See bottom email contact is wrong:


![Image](https://github.com/culturecreates/tech-support-tickets/assets/64489312/e3a9127c-31e8-4bb4-952b-890e3534841e)
